CI/check-format.sh

```diff
@@ -48,7 +48,8 @@ find . -type d \( \
     -path ./plugins/enc-amf -o \
     -path ./plugins/mac-syphon/syphon-framework -o \
     -path ./plugins/obs-outputs/ftl-sdk -o \
-    -path ./plugins/obs-vst \
+    -path ./plugins/obs-vst -o \
```

Comment:
`-path` takes a pattern, so I _think_ specifying `./plugins/obs-vst*` should match all paths that start with the pattern (including subdirs).